### DISCUSSION TITLE
Arduino R4 compatability

### DISCUSSION THREAD

**Discussion by ProgrammingElectronics** [bxparks/AUnit#105] [Q&A]
I am starting a new project using the Arduino R4 WiFi and wanted to use A-Unit during development.

I am getting this error when trying to run the "basic.ino" example"

**#error Platforms using ArduinoCore-API not supported** 

Perhaps this is a simple issue on my end, not sure, any thoughts highly welcomed!

**Comment by bxparks** [comment]
Yeah, the R4 is not supported. I had a discussion about this here:
https://github.com/bxparks/AceCommon/discussions/30
which applies to this project as well.
